### PR TITLE
Improve 4.3.3.

### LINKED ISSUE #2139: edit to Core section 4.3.3
s/element provides navigation to positions in the content that correspond to the locations of page boundaries present in a print source./
element provides navigation to positions in the content that correspond to the locations of page boundaries present in a print source or other static source

### PR DESCRIPTION
**Pull request by iherman**
Fix #2139

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/2154.html" title="Last updated on Mar 31, 2022, 11:00 AM UTC (ad65371)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/2154/d1a3c7f...ad65371.html" title="Last updated on Mar 31, 2022, 11:00 AM UTC (ad65371)">Diff</a>

### CODE REVIEW

File: epub33/core/index.html [hunk 1]
```diff
@@ -4873,7 +4873,7 @@ <h4>The <code>page-list nav</code> Element </h4>
 
 					<p>The <code>page-list</code>
 						<code>nav</code> element provides navigation to positions in the content that correspond to the
-						locations of page boundaries present in a print source.</p>
+						locations of page boundaries present in a print source or other static source.</p>
```
Comment (mattgarrish) [inline epub33/core/index.html]: This paragraph is kind of a mouthful. It also sounds like it excludes page lists without a source.

Maybe something like the following would be better here:

> The `page-list` `nav` element provides navigation to static page boundaries in the content. These boundaries may correspond to a statically paginated source such as print or may be defined exclusively for the EPUB Publication." 

(Any thoughts on this @avneeshsingh @GeorgeKerscher @clapierre @gregoriopellegrino ?)

Comment (mattgarrish) [reply epub33/core/index.html]: Does silence mean everyone is good with this rewrite of the page-list nav description?

Comment (iherman) [reply epub33/core/index.html]: I am fine with it

Comment (iherman) [reply epub33/core/index.html]: (I have updated the PR accordingly.)

**Pull request merged**